gdc_ng_models/models/released_data.py

```diff
@@ -7,9 +7,6 @@
 Base = declarative_base()
 
 
-RELEASE_DATA_TYPE_VALUES = ["ssm", "cnv", "case"]
```

Comment:
Do you think this could be useful to consumers of this module?

Reply:
I hadn't considered that.  I'll add both the value sets as constants.